lib/unison-sqlite/src/Unison/Sqlite/Connection.hs

```diff
@@ -58,26 +58,18 @@ module Unison.Sqlite.Connection
   )
 where
 
-import qualified Data.Text as Text
-import qualified Data.Text.IO as Text
 import qualified Database.SQLite.Simple as Sqlite
 import qualified Database.SQLite.Simple.FromField as Sqlite
 import qualified Database.SQLite3.Direct as Sqlite (Database (..))
 import Debug.RecoverRTTI (anythingToString)
-import System.Environment (lookupEnv)
-import System.IO (stderr)
-import System.IO.Unsafe (unsafePerformIO)
+import Unison.Debug (debugLogM, debugM)
+import qualified Unison.Debug as Debug
 import Unison.Prelude
 import Unison.Sqlite.Exception
 import Unison.Sqlite.Sql
 import UnliftIO (MonadUnliftIO)
 import UnliftIO.Exception
 
-debugTraceQueries :: Bool
-debugTraceQueries =
-  isJust (unsafePerformIO (lookupEnv "UNISON_SQLITE_DEBUG"))
```

Comment:
This is the only spot we were currently using this pattern so I swapped it over.

There are a few other spots using ad-hoc debugging strategies but we can clean those up as we encounter them.